error: argument `unexpected` to `ValidatingUnexpectedArgsUdf` has incompatible type
--> main.sml:1:60
     |
   1 | Ret = ValidatingUnexpectedArgsUdf(required='hi', unexpected=['hello'])
     |                                                             ^ has type `List[str]`, expected `str`